18F/identity-idp
# LG-8990 Remove Post Office phone number

## 🎫 [Ticket](https://cm-jira.usa.gov/browse/LG-8990)

## 🛠 Summary of changes

1. Removed text from en, es and fr translations: "For facility accessibility, use the contact information listed for the Post Office location."
2. Removed 'Contact Information' section from PO search page
3. Removed unused fields from translation yml files
4. Removed phone field from the `usps_locations_controller`
5. Updated tests
6. Removed tty value from interface and PO search and emails templates.
7. Updated email templates
8. Teeny-tiny refactor that is slightly outside of scope: const `LOCATIONS_URL` is repeated, moved it to one location and imported it from there. 


## 📜 Testing Plan

1. Confirm that the text "For facility accessibility, use the contact information listed for the Post Office location." and it's translations are gone from the PO location selection page. 
2. Confirm that Post Office phone number and TTY number are gone, and don't appear anywhere in the IPP flow.
3. Run the usual tests locally and confirm that they pass. 

## 👀 Screenshots - Designers, please offer feedback especially with regards to spacing:

<details>
<summary>After:</summary>

<h3>Translations:</h3>
<img width="567" alt="Screen Shot 2023-03-08 at 3 49 12 PM" src="https://user-images.githubusercontent.com/14644234/224059878-0a33c346-69fb-4273-ab53-d24a93cd283d.png">

<img width="495" alt="Screen Shot 2023-03-08 at 3 49 36 PM" src="https://user-images.githubusercontent.com/14644234/224059882-0bc761a7-f440-4b56-92c1-d1f3a9d2cbf9.png">

<img width="539" alt="Screen Shot 2023-03-08 at 3 50 00 PM" src="https://user-images.githubusercontent.com/14644234/224059884-cc046e5c-04ed-4de4-bed5-3f6d1577918b.png">

<h3>Email:</h3>
<img width="541" alt="Screen Shot 2023-03-08 at 3 51 48 PM" src="https://user-images.githubusercontent.com/14644234/224059887-fcc3a16e-7f91-41bb-852a-d87331e2d890.png">

<h3>Barcode Page:</h3>
<img width="531" alt="Screen Shot 2023-03-08 at 3 52 01 PM" src="https://user-images.githubusercontent.com/14644234/224059889-4fdd9f0d-458d-4493-85b8-1d64d5d0d24b.png">

</details>
